tools/whisper/whisper.xml

```diff
@@ -0,0 +1,322 @@
+            <option value="json">JSON</option>
+            <option value="srt">SubRip</option>
+            <option value="vtt">WebVTT</option>
+            <option value="tsv">Tab-separated values</option>
+        </param>
+        <section name="advanced" title="Advanced Options">
+            <param name="temperature" type="integer" help="temperature to use for sampling" optional="true" value="0" />
+            <param name="best_of" type="integer" help="number of candidates when sampling with non-zero temperature" optional="true" value="5" />
+            <param name="beam_size" type="integer" help="number of beams in beam search, only applicable when temperature is zero" optional="true" value="5" />
+            <param name="patience" type="float" help="optional patience value to use in beam decoding, as in https://arxiv.org/abs/2204.05424, the default (1.0) is equivalent to conventional beam search" optional="true" value="" />
+            <param name="length_penalty" type="float" help="optional token length penalty coefficient (alpha) as in https://arxiv.org/abs/1609.08144, uses simple length normalization by default" optional="true" value="" />
+            <param name="suppress_tokens" type="integer" help="comma-separated list of token ids to suppress during sampling; -1 will suppress most special characters except common punctuations" optional="true" value="-1" />
+            <param name="initial_prompt" type="text" help="optional text to provide as a prompt for the first window" optional="true" value="None" />
+            <param name="condition_on_previous_text" type="boolean" help="if True, provide the previous output of the model as a prompt for the next window; disabling may make the text inconsistent across windows, but the model becomes less prone to getting stuck in a failure loop" truevalue="True" falsevalue="False" checked="true" optional="true" />
+            <param name="temperature_increment_on_fallback" type="float" help="temperature to increase when falling back when the decoding fails to meet either of the thresholds below" optional="true" value="0.2" />
+            <param name="compression_ratio_threshold" type="float" help="if the gzip compression ratio is higher than this value, treat the decoding as failed" optional="true" value="2.4" />
+            <param name="logprob_threshold" type="float" help="if the average log probability is lower than this value, treat the decoding as failed" optional="true" value="-1.0" />
+            <param name="no_speech_threshold" type="float" help="if the probability of the |nospeech| token is higher than this value AND the decoding has failed due to `logprob_threshold`, consider the segment as silence" optional="true" value="0.6" />
+            <conditional name="word_timestamps">
+                <param name="word_timestamps" type="select" label="Word Timestamps" help="(experimental) extract word-level timestamps and refine the results based on them">
```

Comment:
```suggestion
                <param name="word_timestamps" type="select" label="Extract word timestamps" help="(experimental) extract word-level timestamps and refine the results based on them">
```

Maybe that? If you offer true and false you need to ask a question? Maybe Yes, No instead of true, false?